src/op/grad/nn.cc

```diff
@@ -450,6 +450,24 @@ Array<Expr> LayerNormTrainGrad(const Expr& orig_call, const Array<Expr> orig_arg
 
 RAF_OP_GRAD("raf.op.layer_norm_train", LayerNormTrainGrad);
 
+Array<Expr> ReciprocalGrad(const Expr& orig_call, const Array<Expr> orig_args, const Var& y,
+                           const Expr& dy) {
+  static auto op_div = Op::Get("raf.op.divide");
+  static auto op_multiply = Op::Get("raf.op.multiply");
+  static auto op_negative = Op::Get("raf.op.negative");
+  static auto op_ones = Op::Get("raf.op.ones_like");
+  const CallNode* call = orig_call.as<CallNode>();
+  CHECK_GE(call->args.size(), 1);
+  const Expr& x = call->args[0];
+  Call one = Call(op_ones, {x});
+  Call x_pow_2 = Call(op_multiply, {x, x});
+  Call one_divide_x_pow_2 = Call(op_div, {one, x_pow_2});
```

Comment:
How to deal with `x=0`?

Reply:
Outputs are `inf` in this case, same behavior to PyTorch.